src/brs/services/impl/BlockServiceImpl.java

```diff
@@ -107,13 +111,34 @@ public void preVerify(Block block, byte[] scoopData) throws BlockchainProcessor.
     if (block.isVerified()) {
       return;
     }
-
+    
```

Comment:
Code below allow to skip the "nonce" calculation and later checks if the block hash matches. There is no way to change anything on the history without changing this hash. This allow for sync from zero with less than 10% of CPU and in much less time.